Gruntfile.js

```diff
@@ -120,9 +120,25 @@ grunt.initConfig({
 			src: "build/report/lcov/lcov.info"
 		}
 	},
+	concurrent: {
+		target1: [
+			  "build"
+		],
+		target2: [
+			"jshint",
+			"jscs",
+			"search",
+			"qunit"
+		],
+		target3: [
+			"test-on-node"
```

Comment:
instead of numbers I would rename them like:

- target1 => build
- target2 => test
- target3 => node

Also, I'm not sure if this might help, but how about having the qunit task as an individual test, before the test-on-node?